_data/simple-icons.json

```
@@ -2010,6 +2010,11 @@
             "hex": "0F9D58",
             "source": "http://sheets.google.com/"
         },
+        {
+            "title": "Google Street View",
+            "hex": "FFC20B",
+            "source": "https://es.wikipedia.org/wiki/Google_Street_View"
```

Given #2567 I guess we should update this?

```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:Google_Street_View_icon.svg"
```